src/audio/base_fw.c

```diff
@@ -32,6 +33,7 @@ DECLARE_SOF_RT_UUID("basefw", basefw_comp_uuid, 0xe398c32, 0x5ade, 0xba4b,
 DECLARE_TR_CTX(basefw_comp_tr, SOF_UUID(basefw_comp_uuid), LOG_LEVEL_INFO);
 
 static struct ipc4_system_time_info global_system_time_info;
+static uint64_t time_delta;
```

Comment:
```suggestion
static uint64_t host_cycles_delta;
```

Reply:
Don't resolve comments with no answer.

Reply:
uint32_t should be enough, can delta > max(uint32_t), it is 112 seconds.

Reply:
it is not related to precision. Log system use 64bits log so use 64bit